bedrock/settings/static_media.py

```diff
@@ -1326,6 +1333,14 @@
         ),
         'output_filename': 'js/firefox_hello_ie9-bundle.js',
     },
+    'firefox-personal': {
+        'source_filenames': (
+            'js/libs/jquery-1.7.1.min.js',
```

Comment:
Yeah, I was trying to save a few bytes but, turns out a need a few of those files so, makes sense to just include the bundle.